### DISCUSSION TITLE
Time for  a new release?

### DISCUSSION THREAD

**Discussion by troelsarvin** [rustic-rs/rustic#1022] [Q&A] [answered]
Many nice things have gone into rustic-rs since 0.6.1. Could it be time for a new release?
I've been running the 2024-01-24 state of Rustic-rs from Git for a couple of days. It backs up well, and it restores well in my tests.

**Comment by simonsan** [comment]
Yes, but we are waiting for the WebDAV feature to land in rustic_core: https://github.com/rustic-rs/rustic_core/pull/106

**Comment by aawsome** [comment, marked as answer]
@troelsarvin We have just changed the rustic-rs development version (github main) to use the current development version (github main) of rustic_core. We would appreciate very much if you could test this as it contains much more additions. Will also be available after the next nightly run on https://github.com/rustic-rs/nightly

**Comment by troelsarvin** [comment]
Sorry for the late response. Thanks for the new release. Version 0.7.0 (also) works like a charm for me, both for backup and restore.
